src/screens/OrgPost/OrgPost.tsx

```diff
@@ -382,116 +397,54 @@ function orgPost(): JSX.Element {
                 });
               }}
             />
-            {!postformState.postVideo && (
-              <>
-                <Form.Label htmlFor="postPhoto">{t('image')}</Form.Label>
```

Comment:
I think you can delete `image` from `en.json` and other files too. If it's not in use anywhere else in this component/screen.